packages/flutter/pubspec.yaml

```diff
@@ -25,10 +25,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  parse_server_sdk: ^5.1.2
-  # Uncomment for local testing
-  #parse_server_sdk:
-  #   path: ../dart
+  parse_server_sdk: ^5.1.3
```

Comment:
Should we leave this comment for local testing? For example if someone is developing a feature that requires a dart and flutter PR.

Reply:
This code is just an unused comment.
According to our policy, we must keep two packages separate, so we must have separate PR.

Reply:
This comment is for developers who want to develop a feature that includes changes in both packages. When developing such a feature, the contributor likely wants to run both and use the local package. If so, then this comment is helpful for contributors and we should keep it.

Reply:
exactly. This comment was included by me when contributing to the project. It is guidance on how to test changes involving the Dart and Flutter package, as the latter depends on the former. This must be commented on when generating the release.

Reply:
OK, I will return it, but if someone wants to do this, they can change it manually and I think there is no need to comment!

Reply:
remove the comment from pubspec, but add the instruction in another document so that it can guide other contributors. For those who are collaborating with the project it may seem obvious, but for other collaborators who want to contribute for the first time it is not. hence the comment

Reply:
It is a good suggestion, it is better to add it to [CONTRIBUTING.md](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/CONTRIBUTING.md).

Reply:
Let's keep both. We want to make contributions easier, so if someone just has to uncomment, it's easier than guessing where to put the comment, especially for first-time contributors who may not have much experience in this.